validator/src/main.rs

```diff
@@ -1896,6 +1907,7 @@ pub fn main() {
     solana_ledger::entry::init_poh();
     solana_runtime::snapshot_utils::remove_tmp_snapshot_archives(&ledger_path);
 
+    let should_check_duplicate_instance = !matches.is_present("disable_duplicate_instance_check");
```

Comment:
nit:  it's easier to grep around the code if there's internal naming consistency:
```suggestion
    let no_check_duplicate_instance = matches.is_present("no_duplicate_instance_check");
```

Reply:
I think here it would be more consistent to use `should_check_duplicate_instance`.
It is the same name as the argument passed in, and avoid double negation later on, i.e. `! no_ ...`.